staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -29,6 +29,9 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 )
 
+// EtcdFeature is the name of each function that we want to check compatibilitty with etcd.
+type EtcdFeature = string
```

Comment:
IDK if its ok to define a `EtcdFeature` as `string` (its easier to use in `string`) or even define it here.

Reply:
It's part of the storage interface, so it makes sense. One thing to improve is to skip `Etcd` prefix. The storage interface covers also watch cache. In future the might be features handled by watch cache and not by etcd (maybe).

`storage.Feature` would make more sense.